fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -169,14 +170,24 @@ struct TenantManagementWorkload : TestWorkload {
 		}
 	}
 
-	ACTOR Future<Void> _setup(Database cx, TenantManagementWorkload* self) {
+	Future<Optional<Key>> waitDataDbTenantModeChange() const {
+		return runRYWTransaction(dataDb, [](Reference<ReadYourWritesTransaction> tr) {
+			tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+			return tr->get("\xff"_sr); // just a meaningless read
+		});
+	}
+
+	ACTOR static Future<Void> _setup(Database cx, TenantManagementWorkload* self) {
 		Reference<IDatabase> threadSafeHandle =
 		    wait(unsafeThreadFutureToFuture(ThreadSafeDatabase::createFromExistingDatabase(cx)));
 
 		MultiVersionApi::api->selectApiVersion(cx->apiVersion.version());
 		self->mvDb = MultiVersionDatabase::debugCreateFromExistingDatabase(threadSafeHandle);
 
 		if (self->useMetacluster && self->clientId == 0) {
+			std::cout << "Start Meta-cluster and register data cluster..."
+			          << "\n";
+			// Configure the metacluster (this changes the tenant mode)
```

Comment:
```suggestion
			// Configure the metacluster (this changes the tenant mode)
```

Reply:
Actually, I feel these outputs make it clear which step gets wrong when there is an error.

Reply:
In that case, I would change this slighly to:

```suggestion
			fmt::print("Create metacluster and register data cluster\n");
			// Configure the metacluster (this changes the tenant mode)
```